load/load.py

```diff
@@ -12,8 +13,13 @@ def __init__(self, data_store, batch_size=10):
     self.data_store = data_store
     self.batch_size = batch_size
     # Mark data_store as shared memory
-    # Start separate processes for batch_creator and sample_creator(s),
-    # based on the memory config
+    
+    # Start separate processes for sample_creator(s)
+    sc = SampleCreator(self.data_store, 0)
```

Comment:
SampleCreator class does not have a second argument at the time in __init__. What is the `0` for?